dashboard/ontology-extra.ttl

```diff
@@ -0,0 +1,33 @@
+
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix solid: <http://www.w3.org/ns/solid/terms#>.
+@prefix foaf: <http://xmlns.com/foaf/0.1/>.
+@prefix schema: <http:/schema.org/>.
+@prefix ui: <http://www.w3.org/ns/ui#>.
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#>.
+@prefix : <#>.
+
+solid:User a rdfs:Class;
+  rdfs:label "user"@en, "utilisateur"@fr;
+  rdfs:comment """Any person who might use a Solid-based system""";
+  rdfs:subClassOf foaf:Person, schema:Person, vcard:Indivudual.
+
+solid:NewUser a rdfs:Class;
```

Comment:
I think we should use these files instead of the inline versions in basicPreferences.source.ts.

Now we have inconsistent models, as solid:NewUser is commented out in basicPreferences.source.ts.

Reply:
Yes, but the functionality to do that wasn't merged yet in this branch, and I didn't want to rebase again because that caused Tim to locally delete and re-clone this repo locally the previous time. I'll delete the `.ttl` files for now, since they don't do anything currently, and we can recreate them later.